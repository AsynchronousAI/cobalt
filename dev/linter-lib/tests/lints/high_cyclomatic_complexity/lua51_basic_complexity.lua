local cond
local h = function()
    repeat
        print(2)
     until cond()
end

function cond()
    return _G.__PROFILE__
end
return h