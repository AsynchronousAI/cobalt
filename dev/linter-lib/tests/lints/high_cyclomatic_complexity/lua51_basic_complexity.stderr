error[high_cyclomatic_complexity]: cyclomatic complexity is too high (2 > 1)
  ┌─ lua51_basic_complexity.lua:2:11
  │
2 │ local h = function()
  │           ^^^^^^^^^^

